common/hooks/useWindowSize.js

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react';
+import theme from '@weco/common/views/themes/default';
+
+export default function useWindowSize() {
+  const [size, setSize] = useState('large');
+
+  function updateSize() {
+    switch (true) {
+      case window.innerWidth < theme.sizes.medium:
+        setSize('small');
+        break;
+      case window.innerWidth < theme.sizes.large:
+        setSize('medium');
+        break;
+      case window.innerWidth < theme.sizes.xlarge:
+        setSize('large');
+        break;
+      default:
+        setSize('xlarge');
```

Comment:
Given mobile first, should we be working up and defaulting to small?

Reply:
I've changed the initial value to 'small' which is essentially the default - this is just the default of the switch statement.